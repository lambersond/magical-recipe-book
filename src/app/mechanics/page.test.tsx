import { render } from '@test-utils'
import MechanicsPage from './page'

describe('app/mechanics/page', () => {
  it('should match snapshot', () => {
    expect(render(<MechanicsPage />).asFragment()).toMatchSnapshot()
  })
})
