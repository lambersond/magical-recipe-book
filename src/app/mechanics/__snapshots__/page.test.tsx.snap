// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`app/mechanics/page should match snapshot 1`] = `
<DocumentFragment>
  <main
    class="pt-8 pb-20 sm:p-20"
  >
    <div
      class="max-w-6xl mx-auto flex flex-col items-center gap-2 md:gap-4 lg:gap-6"
    >
      <div
        class="bg-card rounded-none sm:rounded-lg shadow-lg p-8"
      >
        <h2
          class="text-3xl font-bold text-secondary mb-6"
        >
          How It Works
        </h2>
        <div
          class="grid md:grid-cols-2 gap-6"
        >
          <div
            class="space-y-4"
          >
            <div
              class="flex items-start space-x-3"
            >
              <div
                class="min-w-8 size-8 bg-secondary rounded-full flex items-center justify-center text-white font-bold"
              >
                1
              </div>
              <div>
                <h3
                  class="font-semibold text-text-primary"
                >
                  Gather Ingredients
                </h3>
                <p
                  class="text-text-secondary"
                >
                  Collect 2 common ingredients and 1 magical component from specific biomes
                </p>
              </div>
            </div>
            <div
              class="flex items-start space-x-3"
            >
              <div
                class="min-w-8 size-8 bg-secondary rounded-full flex items-center justify-center text-white font-bold"
              >
                2
              </div>
              <div>
                <h3
                  class="font-semibold text-text-primary"
                >
                  Cook the Dish
                </h3>
                <p
                  class="text-text-secondary"
                >
                  Make a skill check against the recipe's DC using cooking utensils
                </p>
              </div>
            </div>
          </div>
          <div
            class="space-y-4"
          >
            <div
              class="flex items-start space-x-3"
            >
              <div
                class="min-w-8 size-8 bg-secondary rounded-full flex items-center justify-center text-white font-bold"
              >
                3
              </div>
              <div>
                <h3
                  class="font-semibold text-text-primary"
                >
                  Determine Outcome
                </h3>
                <p
                  class="text-text-secondary"
                >
                  Success grants boons, failure causes detriments
                </p>
              </div>
            </div>
            <div
              class="flex items-start space-x-3"
            >
              <div
                class="min-w-8 size-8 bg-secondary rounded-full flex items-center justify-center text-white font-bold"
              >
                4
              </div>
              <div>
                <h3
                  class="font-semibold text-text-primary"
                >
                  Enjoy Effects
                </h3>
                <p
                  class="text-text-secondary"
                >
                  All effects last for 24 hours
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="grid lg:grid-cols-2 gap-2 md:gap-4 lg:gap-6 w-full"
      >
        <div
          class="bg-card rounded-lg shadow-lg p-6"
        >
          <h2
            class="text-2xl font-bold text-secondary mb-4 flex items-center"
          >
            <span
              class="mr-2"
            >
              üç≥
            </span>
             Cooking Requirements
          </h2>
          <ul
            class="space-y-3 text-text-secondary"
          >
            <li
              class="flex items-start gap-1"
            >
              <span
                class="text-amber-500 mr-2"
              >
                ‚Ä¢
              </span>
              Must have
              <strong
                class="text-text-primary"
              >
                cooking utensils
              </strong>
            </li>
            <li
              class="flex items-start gap-1"
            >
              <span
                class="text-amber-500 mr-2"
              >
                ‚Ä¢
              </span>
              Add 
              <strong
                class="text-text-primary"
              >
                proficiency bonus
              </strong>
               if proficient with cooking utensils
            </li>
            <li
              class="flex items-start gap-1"
            >
              <span
                class="text-amber-500 mr-2"
              >
                ‚Ä¢
              </span>
              Players must commit to eating before rolling
            </li>
            <li
              class="flex items-start gap-1"
            >
              <span
                class="text-amber-500 mr-2"
              >
                ‚Ä¢
              </span>
              All effects last exactly 
              <strong
                class="text-text-primary"
              >
                24 hours
              </strong>
            </li>
          </ul>
        </div>
        <div
          class="bg-card rounded-lg shadow-lg p-6"
        >
          <h2
            class="text-2xl font-bold text-secondary mb-4 flex items-center"
          >
            <span
              class="mr-2"
            >
              üßÇ
            </span>
             Infusing Items
          </h2>
          <p
            class="text-text-secondary mb-4"
          >
            Non-magical consumables can be infused with spell slots to restore hit points:
          </p>
          <div
            class="bg-paper/40 rounded-lg p-4"
          >
            <p
              class="font-semibold text-secondary mb-2"
            >
              Healing: 1d6+4 HP
            </p>
            <p
              class="font-semibold text-secondary mb-2"
            >
              Duration: 7 days
            </p>
            <div
              class="text-sm text-text-secondary space-y-1"
            >
              <p>
                2nd level: 1 item | 3rd level: 3 items
              </p>
              <p>
                4th level: 6 items | 5th level: 10 items
              </p>
              <p>
                6th level: 15 items
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="bg-card rounded-lg shadow-lg p-8 w-full"
      >
        <h2
          class="text-3xl font-bold text-secondary mb-6"
        >
          Gathering Ingredients
        </h2>
        <div
          class="flex flex-wrap gap-8 justify-between"
        >
          <div
            class="flex-1 min-w-64"
          >
            <h3
              class="text-xl font-semibold text-text-primary mb-4"
            >
              Magical Ingredients
            </h3>
            <div
              class="bg-paper/40 rounded-lg p-4 text-text-primary/90"
            >
              <p
                class="font-semibold mb-2"
              >
                Requirements:
              </p>
              <ul
                class="space-y-1 text-text-primary/75"
              >
                <li>
                  ‚Ä¢ Successful Survival check
                </li>
                <li>
                  ‚Ä¢ Must be in correct biome
                </li>
                <li>
                  ‚Ä¢ Magic lasts only 5 days
                </li>
                <li>
                  ‚Ä¢ +2 bonus with PC aid
                </li>
                <li>
                  ‚Ä¢ +1d4 with familiar/companion aid
                </li>
              </ul>
            </div>
          </div>
          <div
            class="flex-1"
          >
            <h3
              class="text-xl font-semibold text-text-primary mb-4"
            >
              Common Ingredients
            </h3>
            <div
              class="bg-paper/40 rounded-lg p-4 text-text-primary min-h-[82%]"
            >
              <p
                class="font-semibold mb-2"
              >
                Requirements:
              </p>
              <ul
                class="space-y-1 text-text-primary/75"
              >
                <li>
                  ‚Ä¢ Spend 1 hour harvesting
                </li>
                <li>
                  ‚Ä¢ Roll 1d6+1 for quantity found
                </li>
                <li>
                  ‚Ä¢ Can be found in any suitable area
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          class="mt-6"
        >
          <h3
            class="text-xl font-semibold text-text-primary mb-4"
          >
            Magical Ingredient Rarities
          </h3>
          <div
            class="flex flex-wrap gap-4 justify-evenly"
          >
            <div
              class="text-text-rarity-common bg-rarity-common/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D4Icon"
                height="24"
                preserveAspectRatio="xMidYMid meet"
                viewBox="0 0 5120 5120"
                width="24"
              >
                <path
                  d="M2349 5024 c-25 -38 -113 -157 -196 -264 -154 -202 -518 -680 -1064 -1400 -174 -228 -450 -593 -615 -810 -389 -512 -474 -626 -474 -639 0 -22 466 -396 1216 -976 50 -38 246 -190 435 -337 189 -146 434 -336 544 -421 l200 -154 6 496 c8 647 8 4026 0 4335 l-6 239 -46 -69z M2710 5080 c-5 -40 -16 -4669 -12 -4833 l7 -227 265 203 c146 112 421 325 611 473 587 457 1227 955 1387 1078 83 65 152 120 152 123 0 9 -566 759 -1465 1943 -157 206 -428 564 -603 794 -367 486 -341 452 -342 446z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Common
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 10
              </p>
            </div>
            <div
              class="text-text-rarity-uncommon bg-rarity-uncommon/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D6Icon"
                height="24"
                viewBox="0 0 100 100"
                width="24"
              >
                <path
                  d="m44.785 98.738c-1.2422-0.69141-5.8477-3.3281-10.238-5.8516-4.3906-2.5273-10.113-5.8047-12.715-7.2891-2.6016-1.4844-6.1836-3.5469-7.9648-4.5898-1.7773-1.043-4.2852-2.4727-5.5703-3.1797l-2.3359-1.2852-0.09375-1.8359c-0.125-2.4492-0.054688-44.891 0.074218-45.152 0.089844-0.18359 0.21094-0.14844 1.168 0.33203 0.96875 0.48828 4.4375 2.4531 8.0117 4.5391 0.75781 0.44141 5.3633 3.0781 10.238 5.8594s11.82 6.7734 15.434 8.875l6.5742 3.8203-0.082032 22.461c-0.046875 12.355-0.12109 22.934-0.16406 23.512l-0.082032 1.0508zm7.7852-2.4258c-0.042968-2.0234-0.078124-12.664-0.078124-23.641v-19.957l4.6992-2.668c9.582-5.4375 25.59-14.613 31.227-17.898 4.7852-2.7891 5.1445-2.9844 5.2812-2.8477 0.32812 0.32812 0.37109 2.668 0.42969 24.465l0.0625 22.453-0.64062 0.36719c-0.35156 0.20312-2.6875 1.5469-5.1914 2.9844s-6.0859 3.4883-7.9648 4.5547c-1.8789 1.0664-5.0977 2.9141-7.1562 4.1094-2.0586 1.1953-5.4805 3.1523-7.6055 4.3477-2.1211 1.1992-4.9102 2.8086-6.1953 3.5781-2.0938 1.2539-6.6289 3.8281-6.7422 3.8281-0.023437 0-0.082031-1.6562-0.125-3.6836zm-4.0898-48.457c-0.59375-0.34766-2.9922-1.7656-5.3281-3.1484-2.3398-1.3828-4.8164-2.8281-5.5078-3.2109-0.69141-0.38281-2.3359-1.3281-3.6523-2.1016-1.3164-0.76953-4.4688-2.5859-7.0039-4.0352-2.5352-1.4492-7.7344-4.4375-11.555-6.6445-3.8203-2.2031-7.043-4.0625-7.1602-4.125-0.17578-0.097656 0.36328-0.45703 2.9922-2.0156 1.7617-1.043 7.8125-4.6289 13.445-7.9688 5.6328-3.3398 12.824-7.5977 15.988-9.4609 3.1602-1.8633 6.1562-3.6445 6.6523-3.957 1.2148-0.76172 2.0859-1.1953 2.4023-1.1953 0.49219 0 4.6016 2.3398 14.594 8.3203 2.9961 1.793 9.2227 5.4844 13.832 8.2031 12.359 7.2891 13.051 7.707 13.09 7.9258 0.050781 0.28516-2.375 1.7227-15.727 9.3242-6.0898 3.4648-13.949 7.9609-22.328 12.773-1.875 1.0742-3.4648 1.9531-3.5312 1.9492-0.070313-0.003906-0.60938-0.28906-1.2031-0.63672z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Uncommon
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 12
              </p>
            </div>
            <div
              class="text-text-rarity-rare bg-rarity-rare/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D8Icon"
                height="24"
                preserveAspectRatio="xMidYMid meet"
                viewBox="0 0 100 100"
                width="24"
              >
                <path
                  d="m52.566 87.938-0.003906-10.469 5.5742-2.6367c3.0664-1.4492 9.6484-4.5586 14.625-6.9102 4.9805-2.3516 10.77-5.0898 12.871-6.0938 4.3984-2.0938 5.9805-2.8086 6.0625-2.7227 0.03125 0.03125-8.7578 8.8867-19.535 19.68l-19.594 19.621-0.003906-10.469zm-16.555-0.67578c-11.527-11.477-27.836-27.953-27.75-28.035 0.074219-0.074218 0.65625 0.1875 4.2969 1.9375 1.8516 0.88672 6.2461 2.9609 9.7695 4.6055s9.8945 4.6484 14.164 6.6758c4.2695 2.0273 8.4375 3.9883 9.2617 4.3555l1.5039 0.67188-0.011718 6.4648c-0.007813 3.5547-0.046875 8.2383-0.085938 10.406l-0.074218 3.9414zm16.633-29.973c-0.046875-7.8672-0.082031-23.617-0.082031-35v-20.691l19.551 19.566c10.754 10.762 21.426 21.469 23.719 23.797l4.168 4.2266-1.582 0.79297c-1.6172 0.80859-28.168 13.402-30.957 14.68-2.625 1.2031-14.273 6.668-14.543 6.8242-0.17578 0.10156-0.20312-1.1875-0.27734-14.195zm-5.9922 14.117c-0.10156-0.039062-2.5391-1.1758-5.418-2.5312s-7.3398-3.4453-9.9141-4.6484c-2.5742-1.2031-7.7852-3.6523-11.578-5.4453-3.793-1.793-9.3359-4.4062-12.316-5.8086-5.8125-2.7344-7.3398-3.5039-7.4219-3.7422-0.058594-0.17969 14.605-15 33.598-33.965l13.695-13.672-0.074219 30.422c-0.070312 29.121-0.15234 39.484-0.3125 39.465-0.039062-0.003907-0.15625-0.039063-0.25781-0.078125z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Rare
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 15
              </p>
            </div>
            <div
              class="text-text-rarity-very-rare bg-rarity-very-rare/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D10Icon"
                height="24"
                preserveAspectRatio="xMidYMid meet"
                viewBox="0 0 100 100"
                width="24"
              >
                <path
                  d="m43.332 96.684c-4.3906-4.2461-9.7461-9.3594-17.293-16.504-8.9297-8.4531-23.906-22.734-24.699-23.547-0.37891-0.39062-0.66406-0.73437-0.62891-0.76562 0.17578-0.17578 20.082-5.8633 20.926-5.9805 0.75781-0.10547 1.5625 0.09375 3.6562 0.90625 2.5664 0.99609 16.805 6.8633 19.43 8.0078l2.2812 0.99609 0.039063 19.312c0.035156 18.098 0.007812 20.906-0.20703 20.891-0.046876-0.003906-1.625-1.4961-3.5039-3.3164zm9.1836-11.527c0-8.0469 0.039063-17.105 0.082031-20.133l0.082032-5.5039 2.3555-0.9375c1.2969-0.51562 5.0625-2.043 8.375-3.3945 3.3086-1.3516 7.8008-3.1758 9.9805-4.0547l3.9609-1.5938 1.8633 0.53125c1.0273 0.29297 5.9609 1.7188 10.969 3.168l9.1016 2.6406-0.35156 0.27734c-0.43359 0.33984-0.73438 0.625-36.801 34.801-6.6328 6.2852-8.2656 7.7891-9.4023 8.6602-0.21094 0.16016-0.22266-0.54688-0.22266-14.457zm-4.625-30.938c-1.6953-0.60938-8.6523-3.4609-13.301-5.457-1.6719-0.71875-3.8945-1.6328-4.9414-2.0312-2.2227-0.85156-3.0898-1.2656-3.1875-1.5234-0.10547-0.27344 0.074218-0.66406 1.9922-4.3008 3.5469-6.7383 16.633-32.105 19.605-38.008 0.70312-1.3945 1.3555-2.6406 1.4531-2.7734 0.16797-0.22266 0.20703-0.19141 0.62891 0.50781 0.70703 1.1719 2.0898 3.8125 8.7422 16.711 3.4141 6.6211 7.7617 15.027 9.6602 18.688 3.5625 6.8672 4.2188 8.2148 4.3359 8.9336 0.0625 0.38672 0.027344 0.44141-0.42188 0.67188-0.89062 0.45312-16.223 6.7734-19.148 7.8945-1.8789 0.71875-3.4609 1.25-3.6953 1.2461-0.10938-0.003906-0.88281-0.25-1.7188-0.55078zm-44.023-5.125c0-0.25 1.4922-1.9375 13.43-15.184 17.168-19.047 19.875-22.016 19.98-21.906 0.0625 0.0625-1.5586 3.3164-3.3203 6.668-0.71484 1.3594-3.3945 6.5195-5.957 11.465-6.2812 12.133-6.875 13.227-7.707 14.098-0.48438 0.51172-0.88672 0.64062-9.5859 3.0742-3.5195 0.98438-6.4961 1.8203-6.6211 1.8555-0.12109 0.035157-0.22266 0.003907-0.22266-0.074218zm90.141-0.22656c-7.0117-1.9531-14.613-4.3594-15.07-4.7695-0.47266-0.42188-1.3555-2.0938-10.27-19.41-1.793-3.4844-3.9336-7.6172-4.7578-9.1836-1.6055-3.0508-1.957-3.8008-1.7891-3.8008 0.13672 0 1.625 1.6211 5.4141 5.8906 1.668 1.8828 4.6875 5.2461 6.707 7.4805 2.0195 2.2344 6.1562 6.8203 9.1875 10.195s6.2812 6.9648 7.2227 7.9766c3.7891 4.0781 5.1953 5.6602 5.1953 5.8516 0 0.17969-0.65625 0.097656-1.8359-0.23047z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Very Rare
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 18
              </p>
            </div>
            <div
              class="text-text-rarity-epic bg-rarity-epic/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D12Icon"
                height="24"
                preserveAspectRatio="xMidYMid meet"
                viewBox="0 0 1200 1200"
                width="24"
              >
                <path
                  d="m411.8 1179.5c-0.28516-0.77344-7.2266-23.906-15.43-51.406s-25.977-87.074-39.496-132.39c-38.004-127.38-50.316-171.08-49.32-175.06 1.7344-6.9062 15.273-15.75 148.79-97.203 26.723-16.305 68.555-41.625 92.953-56.266 50.875-30.531 46.473-29.262 66.562-19.16 17.551 8.8242 134.79 79.727 229.2 138.61 38.324 23.902 46.219 29.312 47.441 32.5 0.45703 1.1953-1.8711 11.469-5.1797 22.828-11.441 39.297-99.91 335.8-100.31 336.19-0.22266 0.21094-84.617 0.92188-187.55 1.5742-124.57 0.79297-187.32 0.71875-187.66-0.21875zm-83.645-38.074c-8.2891-6.1758-38.039-24.301-84.176-51.277l-44.051-25.762-15.98-26.949c-19.25-32.465-49.113-84.277-61.387-106.52-4.918-8.9062-20.789-36.477-35.27-61.266s-40.047-68.691-56.816-97.562l-30.484-52.492 0.72656-151.72 30.281 31.93c16.652 17.562 51.805 54.723 78.109 82.578 113.75 120.46 117.18 124.29 125.3 140.08 8.5039 16.535 12.301 27.906 34.078 102.11 31.941 108.82 52.566 177.91 59.191 198.29 3.4727 10.684 6.1094 20.055 5.8516 20.82-0.25391 0.76562-2.6758-0.25391-5.3789-2.2656zm540.25-11.57c1.1562-3.6797 19.758-65.312 41.34-136.96 21.582-71.652 41.91-137.57 45.168-146.47 9.9492-27.176 15.23-35.902 34.309-56.695 38.07-41.492 126.93-135.55 186.09-196.98l23.941-24.863 0.72656 148.12-11.844 20.492c-66.941 115.83-95.234 164.93-125.83 218.37-41.242 72.027-53.387 91.18-66.398 104.71-11.086 11.531-19.883 17.633-60.352 41.859-33.613 20.121-61.957 35.109-66.395 35.109-2.7617 0-2.7891-0.21875-0.75781-6.6914zm-603.84-374.63c-3.2305-2.9062-30.059-31.277-59.621-63.047-29.562-31.773-80-85.566-112.09-119.55-84.211-89.18-82.984-87.832-82.984-91.262 0-3.2383 19.043-38 46.523-84.934 49.145-83.934 149.82-252.34 151.76-253.86 3.0352-2.3789 12.27-2.2148 25.359 0.44922 35.867 7.3047 293.74 69.684 328.8 79.535l5.2852 1.4844-0.71875 356.49-29.578 18.32c-169.81 105.18-262.74 161.66-266 161.66-0.48437 0-3.5234-2.375-6.7539-5.2812zm654.43 1.8984c-2.3242-1.0234-17.219-9.9688-33.098-19.875-33.312-20.785-180.36-111.24-217.11-133.56-13.676-8.3047-25.879-16.648-27.121-18.539-4.5273-6.9062-6.5547-19.609-7.8789-49.285-1.1992-26.957-2.043-313.37-0.92187-313.37 0.26562 0 47.641-11.434 105.28-25.41 181.97-44.121 232.48-55.91 244.93-57.176l7.7461-0.78516 59.855 101.88c32.922 56.031 78.012 132.6 100.2 170.16l40.348 68.277-15.605 16.945c-148.17 160.91-245.34 262.8-250.48 262.66-1.0586-0.03125-3.8242-0.89453-6.1484-1.918zm-325.34-594.75c-1.5508-0.35547-10.105-2.2383-19.012-4.1836-8.9062-1.9453-49.785-11.672-90.84-21.613-41.055-9.9414-96.191-23.199-122.53-29.461-26.336-6.2617-53.246-12.844-59.801-14.633l-11.914-3.2461 6.2812-4.2617c10.57-7.1719 31.047-18.234 78.23-42.27l45.191-23.02h362.02l37.871 19.812c80.961 42.359 98.375 53.422 84.082 53.422-6.7852 0-72.352 15.352-207.88 48.676-81.746 20.098-93.848 22.57-101.7 20.77z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Epic
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 22
              </p>
            </div>
            <div
              class="text-text-rarity-legendary bg-rarity-legendary/10 rounded-lg p-4 text-center flex-grow"
            >
              <svg
                class="size-8 mx-auto"
                data-testid="D20Icon"
                height="24"
                viewBox="0 0 100 100"
                width="24"
              >
                <path
                  d="m42.07 97.121c-2.7148-1.582-8.8906-5.168-13.719-7.9688-11.559-6.7031-11.246-6.5156-11.023-6.5586 0.23047-0.042969 1.7773 0.21484 14.195 2.3633 5.2305 0.90625 10.801 1.8633 12.379 2.1289 1.5781 0.26562 2.9883 0.53516 3.1406 0.60156l0.27344 0.11719v5.168c0 4.9258-0.070312 7.0391-0.23047 7.0273-0.039063-0.003906-2.2969-1.3008-5.0117-2.8828zm11.152 2.6289c-0.16797-0.27344-0.27344-2.8398-0.30078-7.1836-0.015625-2.4492-0.019531-4.5703-0.011719-4.7148 0.011719-0.22266 0.10938-0.27344 0.65234-0.34375 2.4453-0.30859 11.125-1.7852 20.457-3.4805 6.3672-1.1562 9.0195-1.6055 9.1641-1.5586 0.13672 0.046875-0.96484 0.70312-4.8945 2.9297-4.1562 2.3555-21.91 12.648-23.582 13.672-0.73047 0.44531-1.3438 0.8125-1.3633 0.8125-0.023438 0-0.078125-0.058593-0.12109-0.13281zm-9.2109-18.086c-0.078125-0.074218-6.543-1.2266-16.516-2.9414-4.8984-0.84375-10.465-1.8086-12.379-2.1445-4.3516-0.76562-7.3164-1.207-8.1328-1.207-1.0742 0-1.0625 0.058594-0.34375-1.8086 2.4805-6.4336 12.02-30.336 12.992-32.559 0.41797-0.95703 0.82422-1.7227 0.89844-1.6992 0.23438 0.074218 1.5117 2.2852 8.7773 15.18 1.5195 2.6992 3.1523 5.5469 3.625 6.3281s1.043 1.7852 1.2656 2.2266c0.22266 0.44141 1.875 3.3555 3.668 6.4766 5.9297 10.305 6.8867 11.984 6.8867 12.09s-0.64062 0.15234-0.74219 0.058593zm11.598-0.13281c0-0.10156 0.40234-0.91016 0.89453-1.8008 4.4531-8.0273 22.188-38.918 23.047-40.145l0.25-0.35547 0.35547 0.78516c0.19531 0.42969 1 2.4297 1.7891 4.4414 0.78906 2.0117 2.1758 5.4961 3.082 7.7461 4.1836 10.379 8.8945 22.434 8.8125 22.562-0.19922 0.32422-3.2422 0.90625-16.219 3.1055-10.555 1.793-20.438 3.5156-21.344 3.7266-0.64844 0.14844-0.67188 0.14844-0.67188-0.0625zm-6.1367-2.8359c-1.0469-1.6445-9.2695-15.984-15.332-26.738-1.1914-2.1133-2.5898-4.5547-3.1055-5.4258-0.51953-0.87109-1.1055-1.9141-1.3086-2.3164-0.20312-0.40234-1.1758-2.1328-2.1641-3.8398-0.98828-1.7109-2.0273-3.5273-2.3047-4.043l-0.50781-0.92969 2.8984-0.10156c3.5117-0.125 47.539-0.14453 47.664-0.019531 0.11719 0.11719 0.042969 0.24219-15.77 28.023-8.6055 15.121-9.2461 16.219-9.4297 16.219-0.0625 0-0.35156-0.37109-0.63672-0.82422zm44.934-17.535c-0.92578-2.457-8.9258-22.852-10.176-25.941l-0.42969-1.0664 0.50781-0.41406c1.293-1.0508 6.9727-4.6016 9.3477-5.8398l1.1562-0.60547 0.082031 0.28906c0.10156 0.35547 0.097657 20.984-0.003906 28.883l-0.074219 5.7891-0.41406-1.0938zm-89.348-3.5312c-0.10547-4.9844 0.12109-26.641 0.30078-28.707 0.0625-0.69922 0.16016-1.3203 0.22266-1.3828 0.15625-0.15625 0.92578 0.24609 4.168 2.1914 3.2617 1.957 6.5859 4.1211 6.5859 4.2891 0 0.36328-9.7344 24.957-10.789 27.254l-0.39453 0.86328zm20.777-27.965c0.17969-0.32812 12.465-15.484 14.402-17.773 0.51172-0.60547 1.4219-1.7305 2.0195-2.5 0.59766-0.76953 1.8594-2.3359 2.7969-3.4766 0.94141-1.1406 2.3945-2.9375 3.2305-3.9922 0.83594-1.0547 1.5898-1.9219 1.6719-1.9219 0.17969 0 3.0352 3.3125 5.1562 5.9766 0.82812 1.0391 1.7891 2.2188 2.1367 2.6211s2.0273 2.4609 3.7305 4.5742 4.8828 6.0352 7.0703 8.7188c6.2852 7.7148 6.1406 7.5273 5.918 7.6523-0.48438 0.27344-5.1719 0.33203-25.957 0.33594-21.195 0.003906-22.285-0.007812-22.176-0.21094zm-7.3633-1.1797c-0.26953-0.15625-2.0234-1.1523-3.9023-2.2148-1.8789-1.0625-4.1484-2.3633-5.043-2.8906l-1.6289-0.96484 0.41016-0.27734c1.3164-0.89453 6.1172-3.6094 23.887-13.516 5.3086-2.9609 6.1211-3.3984 6.1797-3.3398 0.070312 0.070312-1.1836 1.668-5.4336 6.9102-11.539 14.242-13.488 16.59-13.793 16.582-0.10156-0.003907-0.40625-0.13281-0.67578-0.28906zm60.82-1.4414c-0.79297-0.95703-2.5938-3.1641-3.9961-4.9102-1.4023-1.7422-2.8828-3.5547-3.2891-4.0234s-1.2578-1.5117-1.8906-2.3164c-0.63281-0.80469-1.8125-2.2578-2.6172-3.2305-3.0625-3.6914-6.1836-7.5781-6.1289-7.6328 0.03125-0.03125 2.3008 1.2188 5.043 2.7734 2.7422 1.5547 9.6523 5.4492 15.355 8.6562 5.7031 3.2031 10.449 5.875 10.551 5.9375 0.14453 0.085937 0.039063 0.1875-0.46875 0.46484-0.35937 0.19531-2.6641 1.4961-5.1211 2.8945-4.9531 2.8164-5.5391 3.1289-5.8047 3.1289-0.10156 0-0.83203-0.78125-1.6289-1.7383z"
                  fill="currentColor"
                />
              </svg>
              <h4
                class="font-bold"
              >
                Legendary
              </h4>
              <p
                class="text-2xl font-bold"
              >
                DC 26
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="bg-card rounded-lg shadow-lg p-8"
      >
        <h2
          class="text-3xl font-bold text-secondary mb-6"
        >
          Cooking Outcomes
        </h2>
        <div
          class="grid lg:grid-cols-3 gap-6"
        >
          <div
            class="bg-red-400/10 rounded-lg p-6 border-l-4 border-red-400"
          >
            <h3
              class="text-xl font-bold text-red-400 mb-3"
            >
              üî• Disastrous
            </h3>
            <p
              class="text-red-500 mb-2 font-semibold"
            >
              Failed DC by 5 or more
            </p>
            <p
              class="text-red-600"
            >
              Causes adverse effects on all who consume the dish. The magical ingredient becomes corrupted, applying the inverse of its intended effect.
            </p>
          </div>
          <div
            class="bg-paper/20 rounded-lg p-6 border-l-4 border-gray-500"
          >
            <h3
              class="text-xl font-bold text-text-secondary mb-3"
            >
              üçΩÔ∏è Edible
            </h3>
            <p
              class="text-text-secondary mb-2 font-semibold"
            >
              Within ¬±4 of the DC
            </p>
            <p
              class="text-text-secondary"
            >
              The dish is edible but provides no magical benefits. It may taste good or bad, but has no special effects.
            </p>
          </div>
          <div
            class="rounded-lg p-6 border-l-4 border-green-400 text-green-400 bg-green-400/10"
          >
            <h3
              class="text-xl font-bold mb-3"
            >
              ‚ú® Divine
            </h3>
            <p
              class="text-green-500 mb-2 font-semibold"
            >
              Exceeded DC by 5 or more
            </p>
            <p
              class="text-green-600"
            >
              Grants the full intended positive effects to all who consume the dish. The magic works perfectly as intended.
            </p>
          </div>
        </div>
      </div>
      <div
        class="bg-card border border-primary rounded-lg p-6"
      >
        <h2
          class="text-2xl font-bold text-secondary mb-4 flex items-center"
        >
          <span
            class="mr-2"
          >
            ‚ö†Ô∏è
          </span>
           Important Notes
        </h2>
        <div
          class="grid md:grid-cols-2 gap-6 text-secondary"
        >
          <div>
            <h3
              class="font-semibold mb-2"
            >
              Magical Ingredient Timing
            </h3>
            <p
              class="text-text-primary"
            >
              Magical ingredients lose their potency after 5 days. Plan your cooking accordingly!
            </p>
          </div>
          <div>
            <h3
              class="font-semibold mb-2"
            >
              Corruption Effects
            </h3>
            <p
              class="text-text-primary"
            >
              When cooking fails dramatically, bonuses become penalties, resistances become vulnerabilities, and advantage becomes disadvantage.
            </p>
          </div>
          <div>
            <h3
              class="font-semibold mb-2"
            >
              Cannot Be Infused
            </h3>
            <p
              class="text-text-primary"
            >
              Magical dishes cannot be further enhanced with spell slot infusion. Their power comes from the cooking process itself.
            </p>
          </div>
          <div>
            <h3
              class="font-semibold mb-2"
            >
              Commitment Required
            </h3>
            <p
              class="text-text-primary"
            >
              Players must decide to eat the dish before the cooking roll is made. No backing out after seeing the result!
            </p>
          </div>
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
